sdk/communication/azure-communication-email/azure/communication/email/_generated/operations/_patch.py

```diff
@@ -6,59 +6,71 @@
+        repeatability_first_sent: str,
+        content_type: str = "application/json",
+        **kwargs: Any
+    ) -> JSON:
         """Queues an email message to be sent to one or more recipients.
 
         Queues an email message to be sent to one or more recipients.
 
-        :param repeatability_request_id: If specified, the client directs that the request is
+        :param email_message: Message payload for sending an email. Is either a model type or a IO
+         type. Required.
+        :type email_message: JSON or IO
+        :keyword repeatability_request_id: If specified, the client directs that the request is
          repeatable; that is, that the client can make the request multiple times with the same
          Repeatability-Request-Id and get back an appropriate response without the server executing the
          request multiple times. The value of the Repeatability-Request-Id is an opaque string
          representing a client-generated, globally unique for all time, identifier for the request. It
          is recommended to use version 4 (random) UUIDs. Required.
-        :type repeatability_request_id: str
-        :param repeatability_first_sent: Must be sent by clients to specify that a request is
+        :paramtype repeatability_request_id: str
+        :keyword repeatability_first_sent: Must be sent by clients to specify that a request is
          repeatable. Repeatability-First-Sent is used to specify the date and time at which the request
          was first created in the IMF-fix date form of HTTP-date as defined in RFC7231. eg- Tue, 26 Mar
          2019 16:06:51 GMT. Required.
-        :type repeatability_first_sent: str
-        :param email_message: Message payload for sending an email. Required.
-        :type email_message: IO
-        :keyword content_type: Body Parameter content-type. Content type parameter for binary body.
-         Default value is "application/json".
+        :paramtype repeatability_first_sent: str
+        :keyword content_type: Body Parameter content-type. Known values are: 'application/json'.
+         Default value is None.
         :paramtype content_type: str
-        :return: SendEmailResult or the result of cls(response)
-        :rtype: ~azure.communication.email.models.SendEmailResult
+        :return: JSON object
+        :rtype: JSON
         :raises ~azure.core.exceptions.HttpResponseError:
         """
+
         message_id = super().send(
-            repeatability_request_id,
-            repeatability_first_sent,
             email_message,
-            **dict(kwargs, cls=self.__return_message_id)
+            repeatability_request_id=repeatability_request_id,
+            repeatability_first_sent=repeatability_first_sent,
+            content_type=content_type,
+            cls=self.__return_message_id,
+            **kwargs
         )
-        return SendEmailResult(message_id=message_id)
-    
+
+        return { "message_id": message_id }
```

Comment:
Looks like we should make this `{"messageId": message_id}` to be consistent with the response from `get_status`